src/yggdrasil/nodeinfo.go

```diff
@@ -108,9 +108,14 @@ func (m *nodeinfo) setNodeInfo(given interface{}) error {
 	if nodeinfomap, ok := given.(map[string]interface{}); ok {
 		for key, value := range nodeinfomap {
 			if _, ok := newnodeinfo[key]; ok {
+				if value == "hide" {
+					delete(newnodeinfo, key)
+				}
 				continue
 			}
-			newnodeinfo[key] = value
+			if value != "hide" {
```

Comment:
maybe this could be moved up and folded into one check overally, something like `if s, ok := value.(string); ok && strings.EqualFold(s, "hide") { delete(newnodeinfo, key); continue }`